### ISSUE TITLE
Request: spawn location

### ISSUE THREAD

**Issue by Chriscox462** [zSupremoz/Paper-Mario-The-Origami-King-Save-Editor#5]
Would it be possible to change where the game starts out when you load a save?

**Comment by NoOneOfSignificance** [comment]
Yes. Changing your load location is impossible and I plan on adding it in an update I plan on releasing very soon.
